content/applications/sales/crm/optimize/partner_autocomplete.rst

```diff
@@ -1,48 +1,82 @@
-===================================================
-Enrich your contacts base with Partner Autocomplete
-===================================================
+=========================================
+Enrich contacts with partner autocomplete
+=========================================
 
-Parter Autocomplete helps you to enrich your contacts database with corporate data. Select one of
-the companies suggested in the dropdown, and quickly get all the information you need.
+*Partner autocomplete* enriches the contacts database with corporate data. In any module, enter the
+new company name into the :guilabel:`Customer` field (partner_id technical field), and select one of
+the companies suggested in the drop-down menu. Instantly get valuable company information full of
+hard-to-find data for a desired company.
+
+.. important::
+   A company **cannot** already be manually entered in the *Contacts* application prior to enriching
+   it with data.
+
+The information provided by partner autocomplete can include general information about the business
+(including full business name and logo), social media accounts, :guilabel:`Company type`,
+:guilabel:`Founded` information, :guilabel:`Sectors` information, the number of
+:guilabel:`Employees`, :guilabel:`Estimated revenue`, :guilabel:`Phone` number,
+:guilabel:`Timezone`, and :guilabel:`Technologies Used`.
+
+.. important::
+   When getting a company's contact information make sure to be aware of the latest EU regulations.
+   For more information about General Data Protection Regulation refer to: `Odoo GDPR
+   <http://odoo.com/gdpr>`__. In Odoo, individual contact information cannot be searched for with
```

Comment:
External links only need a [single underscore](https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#use-single-underscore-suffixes-for-hyperlink-references)
```suggestion
   <http://odoo.com/gdpr>`_. In Odoo, individual contact information cannot be searched for with
```